resources.go

```diff
@@ -2455,3 +2497,13 @@ func updateLegacyModuleNames(prov *tfbridge.ProviderInfo) {
 	updateModuleName("aws_ses_configuration_set", "ConfgurationSet", "ConfigurationSet",
 		sesMod, sesMod, nil)
 }
+
+func TransformWithRandomString(n int) string {
```

Comment:
Does this need to be exported? Or if it is used in multiple places can it go in the bridge instead?

Reply:
Fixed!